src/cbrkit/sim/collections.py

```diff
@@ -148,8 +182,56 @@ def compute_dtw(self, x: np.ndarray, y: np.ndarray) -> float:
                     )
                     dtw_matrix[i, j] = cost + last_min
 
-            return dtw_matrix[n, m]
-
+            # Backtracking to find the best alignment
+            alignment = self.backtrack(dtw_matrix, x, y, n, m)
```

Comment:
This should be optional as it requires additional computations that are not needed in many cases. If disabled, the alignment could be `None`